### ISSUE TITLE
Add bulk action to get full object cocina

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/argo#3847]
Sometimes you need the full cocina for a set of objects in the original JSON because the flattening effect of CSV transformations makes a CSV version too hard to parse.

**Comment by justinlittman** [comment]
What do you want the export format to be? Line-oriented JSON (my suggestion)? Zip?

**Comment by andrewjbtw** [comment]
Line-oriented JSON is fine but given the size of some requests I think I might file, may be good to have it come out gzipped.

**Comment by lwrubel** [comment]
@andrewjbtw where should this item be listed in the Bulk Actions dropdown menu? Under "Generate reports"? And what should it be called--"Download Cocina JSON"?

**Comment by andrewjbtw** [comment]
> Under "Generate reports"?

Yes.

> "Download Cocina JSON"

I have a bit of concern that people don't always understand the difference between "Cocina" and "Cocina descriptive metadata". What about "Download full Cocina JSON"?
